### PR TITLE
Fix Lucida Grande is always used for release notes

### PR DESCRIPTION
**Pull request by xhacker**
Use "-apple-system-font" to use system default font.

### CODE REVIEW

File: Sparkle/SUUpdateAlert.m [hunk 1]
```diff
@@ -107,15 +107,10 @@ - (IBAction)remindMeLater:(id)__unused sender
...
-    // Set the default font
     [self.releaseNotesView setPreferencesIdentifier:SUBundleIdentifier];
     WebPreferences *prefs = [self.releaseNotesView preferences];
-    NSString *familyName = [[NSFont systemFontOfSize:8] familyName];
-    if ([familyName hasPrefix:@"."]) { // 10.9 returns ".Lucida Grande UI", which isn't a valid name for the WebView
-        familyName = @"Lucida Grande";
-    }
-    [prefs setStandardFontFamily:familyName];
-    [prefs setDefaultFontSize:(int)[NSFont systemFontSizeForControlSize:NSSmallControlSize]];
+    [prefs setStandardFontFamily:@"-apple-system-font"];
```
Comment (jakepetroules) [inline Sparkle/SUUpdateAlert.m]: Not `-apple-system`? (as implied by http://trac.webkit.org/changeset/185750)

Comment (xhacker) [inline Sparkle/SUUpdateAlert.m]: You are right `-apple-system` is the new recommended token. But it was added recently (https://bugs.webkit.org/show_bug.cgi?id=144484), so I believe we can’t use it in older systems.

Comment (jakepetroules) [inline Sparkle/SUUpdateAlert.m]: OK... please add a comment explaining the `-apple-system` and `-apple-system-font` tokens and why we're using it.

Comment (xhacker) [inline Sparkle/SUUpdateAlert.m]: Updated. Also did a little refactoring.

**Pull request merged**